### PR TITLE
Added myself to the team site.

### PR DESCRIPTION
**Pull request by berithpy**
This PR isn't ready yet, i'm waiting on someone to proofread my bio!

### CODE REVIEW

**Comment by JessicaJackson** [comment]
First review done in email. Needs another review after updates are made.

File: source/team/ddiaz.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+  date: 24/01/2018
```
Comment (german-e-mas) [inline source/team/ddiaz.html.erb]: Can you modify the date to appear below Emi but above Fer? Set this date to Fer and use his actual date value for you.

File: source/team/ddiaz.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  tags: team
+---
+
+<%= image_tag("team/ddiaz-thumbnail.jpg", :alt => "Diego",
```
Comment (german-e-mas) [inline source/team/ddiaz.html.erb]: Have you optimized the thumbnail with the guidelines of the [Dev notes](https://github.com/ekumenlabs/ekumenlabs.github.io/blob/middleman/DEV-NOTES.md)?

File: source/team/ddiaz.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      He enjoys developing web applications in Python and Javascript and has also been working with cloud hosted containers with kubernetes.
+    </p>
+    <p>
+      He is into IOT and hopes someday everything is connected.
+    </p>
+    <p>
+      In his free time he participates regularly in Game Jam (Gamedev hackathons), cooks, plays the bass, games and runs a tiny 3D printing company.
+    </p>
+  </div>
+</div>
```
Comment (german-e-mas) [inline source/team/ddiaz.html.erb]: nit: Add a new line at the end of the file.

File: source/team/ddiaz.html.erb (hunk 2 above)
Comment (berithpy) [reply source/team/ddiaz.html.erb]: No i didn't will do!

**Comment by berithpy** [comment]
@german-e-mas yes! The readme is great!

**Comment by german-e-mas** [comment]
Thanks @berithpy ! LGTM. Let me merge and deploy.

**Comment by german-e-mas** [comment]
Merging. Thanks @berithpy for the PR and @JessicaJackson for the review!

**Comment by gecofernando** [comment]
I'd really like to re-crop the pic to get a more similar proportion to the others.
Could you share it with me so I do it when I do FabriB's, Istvan's and Lea's?

**Comment by german-e-mas** [comment]
Good point @gecofernando . Can you send him the non-cropped picture, @berithpy ?

Please open an issue when you have them (along with the text).

**Comment by berithpy** [comment]
@gecofernando ok i'll send it your way!

**Pull request merged**